src/GraphQL/ResolverRegistryInterface.php

```diff
@@ -46,6 +46,11 @@ public function addFieldResolver($type, $field, ResolverInterface $resolver);
    */
   public function getFieldResolver($type, $field);
 
+  /**
+   * @return callable
+   */
+  public function getAllFieldResolvers();
```

Comment:
GraphQL 4.0 has been released, so not sure if we should change the interface at this point.

Would it be enough for you to just implement this method on the ResolverRegistry class?